### ISSUE TITLE
Emoji detection breaks MAC addresses

### ISSUE THREAD

**Issue by ArcturusB** [wekan/wekan#1248]
**Impacted version**: v0.40.0 and lower

**Problem description**
Emoji detection tries to handle all patterns like `:foo:`, even when it shouldn't. This leads to broken MAC addresses.

**Example**
~~~
01:02:03:04:05:06

`01:02:03:04:05:06`

```
01:02:03:04:05:06
```
~~~

produces

![wekan_mac_emoji_bug](https://user-images.githubusercontent.com/9514702/30800812-8bf8e56a-a1e1-11e7-96da-32f69da12017.png)

GitHub seems to handle this by not trying to display images for emojis that do not exist, and not replacing `:foo:`-like sequences in code environments. Maybe Wekan could do the same?

**Comment by xet7** [comment]
@ArcturusB

I do have code to disable emojis, this would also fix related issues #743 #643 #323 where some others have asked a way to disable emojis. Some other emoji packages like emojione also break urls etc. Emojis could be added back sometime if somebody finds a way to use them without breaking something.

**Comment by ocdtrekkie** [comment]
This issue was not fixed in 0.55 on Sandstorm.

![image](https://user-images.githubusercontent.com/4399499/33290129-620e058c-d387-11e7-9c17-d393326941d5.png)
